recipes/devel/mtools.yaml

```diff
@@ -0,0 +1,24 @@
+inherit: [autotools]
+
+metaEnvironment:
+   PKG_VERSION: "4.0.36"
```

Comment:
Could you please merge this with https://github.com/BobBuildTool/basement-gnu-linux/pull/32/commits/249eb25978e031f282bef3bd6598b43ec60e23d5? I'm also fine with merging @sixtyfourktec's version if you prefer that. (not sure if the libiconv dependency is crucial)